packages/jwt-service/__tests__/localAgent.test.ts

```diff
@@ -22,7 +22,7 @@ let dbConnection: OrPromise<DataSource>
 let agent: TAgent<IKeyManager & IDIDManager & IIdentifierResolution & IJwtService>
 
 const DID_METHOD = 'did:jwk'
-// const PRIVATE_KEY_HEX = '7dd923e40f4615ac496119f7e793cc2899e99b64b88ca8603db986700089532b'
+119f7e793cc2899e99b64b88ca8603db986700089532b'
```

Comment:
Whats up here?

Reply:
Fixed already.
Bram requested to remove these, somehow IJ must have messed up my selection